### PR TITLE
mpg123 => 1.29.3

### PR DESCRIPTION
**Pull request by saltedcoffii**
Works on x86_64. needs binaries.

```
CREW_TESTING_REPO=https://github.com/saltedcoffii/chromebrew.git CREW_TESTING_BRANCH=mpg123_1.29.3 CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/mpg123.rb [hunk 1]
```diff
@@ -1,39 +1,28 @@
...
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mpg123/1.25.10_x86_64/mpg123-1.25.10-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '25786bf03b27a76815ab30e8f3c8e864e7f2f58548b84e092e93f51a074e2515',
-     armv7l: '25786bf03b27a76815ab30e8f3c8e864e7f2f58548b84e092e93f51a074e2515',
-       i686: 'efe58119727f1d513309661bc643250ce3dff4dfd313a7e8e0b8b3bbdba00baf',
-     x86_64: '28edfc1ab64e472a915f6561cb810ed4f69a6ec06d38d7209cef400b823d9328',
-  })
-
-  depends_on 'alsa_plugins'
```
Comment (uberhacker) [inline packages/mpg123.rb:25]: Why were alsa plugins removed?

Comment (saltedcoffii) [reply packages/mpg123.rb:25]: mpg123 doesn't link against anything from alsa_plugins, only things from alsa_lib

File: packages/mpg123.rb [hunk 2]
```diff
@@ -1,39 +1,28 @@
...
-  depends_on 'cras'
+  depends_on 'alsa_lib'
+#  depends_on 'cras'
 
   def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--with-default-audio=oss'
+    system "#{CREW_ENV_OPTIONS} ./configure #{CREW_OPTIONS} \
```
Comment (supechicken) [inline packages/mpg123.rb:16]: We don't need to add `CREW_ENV_OPTIONS` to the command now (it will be inserted by package.rb automatically now)

Comment (uberhacker) [reply packages/mpg123.rb:16]: If `CREW_ENV_OPTIONS` is added does only one set of config options get added?

Comment (supechicken) [reply packages/mpg123.rb:16]: The newly passed `CREW_ENV_OPTIONS` in the command will cover it (kind of redefine) (so there is always one CFLAGS only)

**Pull request merged**